### DISCUSSION TITLE
Change Page Orientation for Multi-Language User

### DISCUSSION THREAD

**Discussion by WeiKok** [sumatrapdfreader/sumatrapdf#2915] [Ideas]
I have been using Sumatra for many years, and being in Asia meant I have to deal with PDFs/CBRs that are left-to-right or right-to-left all the time.

I understand that I can change CBR orientation (CbxMangaMode) in Advanced Options, but if I have to read two documents in different languages simultaneously, I can't. Also, using Advanced Options meant I had to restart Sumatra whenever I changed direction. Besides, there is no option for PDF.

My suggestion is simple. Why not create an icon on the toolbar that allows us to switch with just a click/touch without restarting?

Most importantly, none of the readers I know has that function on the toolbar. You could be the first =)

**Comment by GitHubRulesOK** [comment]
hmm somewhere I use Shift+M for flip R2L with L2R does that not work on a per page basis, but its manual shortcut not mouse icon see
https://github.com/sumatrapdfreader/sumatrapdf/discussions/2775#discussioncomment-3066563

![image](https://user-images.githubusercontent.com/12754913/187894437-a12bec83-833b-4f27-a4ea-38718a0ceaba.png)
